code/workspaces/auth-service/src/dataaccess/userRolesRepository.js

```diff
@@ -28,16 +28,28 @@ function addDefaults(roles) {
 async function getRoles(userId, userName) {
   let roles = await UserRoles().findOne({ userId }).exec();
   if (!roles) {
-    roles = await addRecordForNewUser(userId, userName);
+    roles = await createNonVerifiedUserRecord(userId, userName);
   }
 
   return addDefaults(roles);
 }
 
+// User has logged in for the first time, set up userRoles database record if does not exist
+// and set the user as verified
+async function createNonVerifiedUserRecord(userId, userName) {
```

Comment:
@iwalmsley Unless I'm misunderstanding - I think createNonVerifiedUserRecord() and createVerifiedUserRecord() should have their names switched - they are doing the exact opposite of what their name says.  This will impact the code that uses them, which does work at the moment, but must read a bit odd?